pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+## Problem
+Why are we making this code change?
+
+## Solution
+How do the changes in this pull request solve the stated problem? Be descriptive.
+
+## Additional Notes
+Is there anything in particular that you want to call attention to? Areas of focus, follow-up actions, etc.
+
+## Links
+### Ticket
+- [ ] GitHub issue _____
+### Other links
+
+## Verification
+### Manual tests
+- [ ] Include the data and actions taken to exercise the Subject Under Test (SUT). Include any screen captures if relevant.
```

Comment:
Doesn't really make sense for this to be a checkbox?

Reply:
removed all the extra checkboxes